### PR TITLE
Issue #11 : Add Support for `FrameLayout`

### PR DESCRIPTION
**Pull request by jerryOkafor**
Adds basic support for `FrameLayout` using Box.

### CODE REVIEW

File: recompose-composer/src/main/kotlin/recompose/composer/visitor/ComposingVisitor.kt [hunk 1]
```diff
@@ -171,6 +172,14 @@ internal class ComposingVisitor : Visitor {
         }
     }
 
+    override fun visitFrameLayout(node: FrameLayoutNode) {
+        val composable = "Box"
+
+        writer.writeCall(composable) {
+            node.viewGroup.children.forEach { it.accept(this@ComposingVisitor) }
+        }
+    }
```
Comment (pocmo) [inline recompose-composer/src/main/kotlin/recompose/composer/visitor/ComposingVisitor.kt:181]: nit

**Review by pocmo — APPROVED** [review]
👍

File: recompose-composer/src/main/kotlin/recompose/composer/visitor/ComposingVisitor.kt [hunk 2]
```diff
@@ -171,6 +172,14 @@ internal class ComposingVisitor : Visitor {
         }
     }
 
+    override fun visitFrameLayout(node: FrameLayoutNode) {
+        val composable = "Box"
+
+        writer.writeCall(composable) {
```
Comment (pocmo) [inline recompose-composer/src/main/kotlin/recompose/composer/visitor/ComposingVisitor.kt:178]: Let's maybe use the more verbose version here, like in the other methods. And let's not forget about adding the modifiers!

```Kotlin
val modifier = ModifierBuilder(node)

writer.writeCall(
    name = "Box",
    parameters = listOf(
        modifier.toCallParameter()
    )
) {
   // ...
}
```

**Comment by pocmo** [comment]
There's a conflict here now after merging another PR.

File: recompose-composer/src/main/kotlin/recompose/composer/visitor/ComposingVisitor.kt (hunk 2 above)
Comment (jerryOkafor) [reply recompose-composer/src/main/kotlin/recompose/composer/visitor/ComposingVisitor.kt:178]: I will make the changes and update my local.

**Comment by jerryOkafor** [comment]
@pocmo we are good to go.

**Review by pocmo — APPROVED** [review]
👍

**Pull request merged**